lib/PuppeteerSharp.Tests/ElementHandleTests/ScreenshotTests.cs

```diff
@@ -150,7 +150,25 @@ public async Task ShouldNotHangWithZeroWidthHeightElement()
             await Page.SetContentAsync(@"<div style='width: 0; height: 0'></div>");
             var elementHandle = await Page.QuerySelectorAsync("div");
             var screenshot = await elementHandle.ScreenshotDataAsync();
-            Assert.NotNull(screenshot); 
+            Assert.NotNull(screenshot);
+        }
+
+        [Fact]
+        public async Task ShouldWorkForAnElementWithFractionalDimensions()
+        {
+            await Page.SetContentAsync("<div style=\"width:48.52px;height:19.8px;border:1px solid black;\"></div>");
```

Comment:
notice this is 48.52 and not 48.51 like puppeteer
for some reason when using 48.51 it was parsed as 48.50 and `Math.Round` rounds it to 48 and not 49